gometalinter/gometalinter-tap

```diff
@@ -23,12 +23,19 @@ main() {
 		gometalinter_args+=(--config "${config_file}")
 	fi
 
+	echo '~~~ Preparing code for $GOPATH'
+	mkdir -p "${GOPATH}/src/github.com/${TEAMCI_REPO_SLUG}"
+	cp -vr . "${GOPATH}/src/github.com/${TEAMCI_REPO_SLUG}"
```

Comment:
@antonlindstrom Is it safe to assume that most projects are developed in this directory structure? I mean, your development work happens inside the `$GOPATH/src/github.com/mycompany/repo`directory.

Reply:
I think this will cover 99% of the cases :+1: